htdocs/product/index.php

```diff
@@ -347,7 +347,7 @@
 // TODO Move this into a page that should be available into menu "accountancy - report - turnover - per quarter"
 // Also method used for counting must provide the 2 possible methods like done by all other reports into menu "accountancy - report - turnover":
 // "commitment engagment" method and "cash accounting" method
-if ($conf->global->MAIN_FEATURES_LEVEL)
+if ($conf->global->MAIN_SHOW_PRODUCT_ACTIVITY_TRIM)
```

Comment:
MAIN_FEATURES_LEVEL is often define, but the new constant may not. To avoid warning can you replace with 
if (! empty($conf->global->MAIN_SHOW_PRODUCT_ACTIVITY_TRIM))